build.sbt

```diff
@@ -25,6 +31,22 @@ def jsonDependencies(scalaVersion: String) = Seq(
   logback % Test
 ) ++ jacksons ++ specsBuild.map(_ % Test)
 
+// Common settings 
+import com.typesafe.sbt.SbtScalariform._
+import scalariform.formatter.preferences._
+
+val previousVersion = "2.6.0-M1" // first from this separate repo
```

Comment:
Maybe a better approach here is use something like play-slick. See https://github.com/playframework/play-slick/blob/6506de9228e17961cf121cee1b30044c9bf74920/build.sbt#L44-L52

Of course, after releasing a version and setting a branch for it, we should set the `previousVersion` variable to _activate_  MiMa.